tests/wpunit/ThemeObjectQueriesTest.php

```diff
@@ -51,27 +56,29 @@ public function testThemeQuery() {
 		/**
 		 * Run the GraphQL query
 		 */
+		wp_set_current_user( $this->admin );
 		$actual = do_graphql_request( $query );
 
 		$screenshot = $actual['data']['theme']['screenshot'];
 		$this->assertTrue( is_string( $screenshot ) || null === $screenshot );
 
+		$theme = wp_get_theme( $theme_slug );
 		/**
 		 * Establish the expectation for the output of the query
 		 */
 		$expected = [
 			'data' => [
 				'theme' => [
-					'author'      => null,
+					'author'      => $theme->author,
 					'authorUri'   => 'https://wordpress.org/',
-					'description' => null,
```

Comment:
I think this was previously broken, couldn't get this data at all with the old query 🤷‍♂️

Reply:
🤔 interesting. . .this _looks_ like a better test anyway!